fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -2579,6 +2582,12 @@ ACTOR Future<Void> reply(CommitBatchContext* self) {
 		                                     self->prevVersion,
 		                                     writtenTags),
 		    TaskPriority::ProxyMasterVersionReply));
+		if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
+			ASSERT(self->loggingComplete.isReady());
+			ASSERT_WE_THINK(self->loggingComplete.get() >= self->commitVersion); // @todo remove later
+			pProxyCommitData->minKnownCommittedVersion =
+			    std::max(pProxyCommitData->minKnownCommittedVersion, self->loggingComplete.get());
```

Comment:
To be totally correct, I think we need to check "if (!self->loggingCompete.isError())" before invoking "self->loggingComplete.get()".

Reply:
The "wait()" here https://github.com/apple/foundationdb/blob/5497488e411735beb1e52f3ee559cd17fa2258f8/fdbserver/CommitProxyServer.actor.cpp#L2504 would probably throw an error if "self->loggingCompete" got set to an error state, so probably the "isError()" check is not needed here. I can add it if we think it is safer to have that check.